src/job_queue/job_result_wrapper.rs

```diff
@@ -0,0 +1,267 @@
+//! This module provides type `JobResultWrapper` to enhance the ergonomics of
+//! working with job-specific result types which must implement the `JobResult`
+//! trait.
+//!
+//! It is useful for:
+//!
+//! 1. returning job results of type T as `Box<dyn JobResults>` when implementing
```

Comment:
Once here, once below:

```suggestion
//! 1. returning job results of type T as `Box<dyn JobResult>` when implementing
```

Reply:
addressed in 8b189964d2c0fd51b8e9c4f7899b2f713bc95b99